### PR TITLE
Alex's hw

### PR DESCRIPTION
**Pull request by alexfigliolia**
Hey guys, I did the homework, but found myself interested in the concise answers of the solution branch. I walked myself through your solutions compared to my own as well as the ones I could not figure out on my own.

### CODE REVIEW

File: src/callbacks.js [hunk 1]
```diff
@@ -69,13 +97,27 @@ contains(foods, 'ribeye', (result) => {
 // Pass the array to the callback function.  Do not mutate the original array.
 // code here
 
+const removeDuplicates = (arr, cb) => {
+	let unique = [];
+	arr.forEach((item, i) => {
+		if(unique.indexOf(item) < -1) unique.push(item);
```
Comment (sperrye) [inline src/callbacks.js:103]: You may want `===` or `< 0`  instead of `< -1`.

**Review by sperrye — COMMENTED** [review]
Completed review, great work

File: src/callbacks.js (hunk 1 above)
Comment (alexfigliolia) [reply src/callbacks.js:103]: Hey Emily! Are you sure? I believe Array.prototype.indexOf returns -1 if the item is not in the array  - or the index of the item if it does. If it returns zero then the tested item is present at index 0.

Comment (alexfigliolia) [reply src/callbacks.js:103]: I probably should have used !== -1 ?

**Pull request closed without merging**